File: adminpro-angular7-lite/Angular/src/app/dto/disableEncuesta.dto.ts
export class DisableEncuestaDto {
    encuesta: boolean;
    id: string;
    email: string
    password: string;

    constructor(encuesta: boolean, id: string, email: string, password: string){
        this.encuesta=encuesta;
        this.id=id;
        this.email=email;
        this.password=password;
    }
}

File: ExpoFpApp/app/src/main/java/com/example/expofpapp/Model/DisableEncuestaDto.java
package com.example.expofpapp.Model;

public class DisableEncuestaDto {
    private Boolean encuesta;

    public DisableEncuestaDto(Boolean encuesta) {
        this.encuesta = encuesta;
    }

    public Boolean getEncuesta() {
        return encuesta;
    }

    public void setEncuesta(Boolean encuesta) {
        this.encuesta = encuesta;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;

        DisableEncuestaDto that = (DisableEncuestaDto) o;

        return encuesta != null ? encuesta.equals(that.encuesta) : that.encuesta == null;
    }

    @Override
    public int hashCode() {
        return encuesta != null ? encuesta.hashCode() : 0;
    }

    @Override
    public String toString() {
        return "DisableEncuestaDto{" +
                "encuesta=" + encuesta +
                '}';
    }
}
